GoogleForCreators/web-stories-wp
# Dashboard: Story Previews

_Requires #3390 to be merged first_
## Summary
Adds ability to preview a story from the dashboard

## Relevant Technical Choices
- Builds off of foundation set in #3390 
- Swaps out feature flags for the center action on 'my stories' to tie it with the new `enableStoryPreviews` flag 
- Condenses reducer and actions associated with assembling markup for preview since it turns out stories need the same assembly of props that templates do.

## To-do
- Do we want the ability to preview through the list view? 

## User-facing changes
- If you enable the `enableStoryPreviews` flag you should be able to open a preview of a story by clicking on a story preview or the center button (view or preview) of the story when you hover/focus. Same goes for hitting 'enter'. 

## Testing Instructions
- Enable the `enableStoryPreviews` feature flag
- Navigate to the dashboard
- click on a story or the 'view'/'preview' button of a story and see that that story's preview opens in the same window as the dashboard. 
![preview story](https://user-images.githubusercontent.com/10720454/89592392-5a504b80-d801-11ea-9733-c1535428c7be.gif)


---

Addresses #3391
